sdk/cognitiveservices/cognitiveservices-customvision-training/src/trainingAPIClient.ts

```diff
@@ -519,25 +518,25 @@ class TrainingAPIClient extends TrainingAPIClientContext {
    * @summary Update metadata of images.
    * @param projectId The project id.
    * @param imageIds The list of image ids to update. Limited to 64.
-   * @param metadata The metadata to be updated to the specified images. Limited to 50 key-value
-   * pairs per image. The length of key is limited to 256. The length of value is limited to 512.
+   * @param metadata The metadata to be updated to the specified images. Limited to 10 key-value
+   * pairs per image. The length of key is limited to 128. The length of value is limited to 256.
```

Comment:
Wont this change in limit be a breaking change for the users? Or is it the case that things were wrongly documented before?

Reply:
cc @areddish

Reply:
@ramya-rao-a  Yes, this is a documentation only fix, nothing is changing in the service. We released with the lower limits, so this isn't breaking.